lua/telescope/builtin/internal.lua

```diff
@@ -31,12 +31,9 @@ end
 
 local internal = {}
 
--- TODO: What the heck should we do for accepting this.
---  vim.fn.setreg("+", "nnoremap $TODO :lua require('telescope.builtin').<whatever>()<CR>")
--- TODO: Can we just do the names instead?
 internal.builtin = function(opts)
-  opts.path_display = utils.get_default(opts.path_display, "hidden")
```

Comment:
This one was causing files to appear with only an icon, and I didn't see it in the docs anyway, so I removed it.

Reply:
We dont do anything with `path_display` and `ignore_filename` so you can remove both.